<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>first_window_for_user</class>
 <widget class="QDialog" name="first_window_for_user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>11</y>
     <width>31</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>⚙</string>
   </property>
  </widget>
  <widget class="QWidget" name="sidewidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>200</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>530</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>退出登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>个人主页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reserveBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>图书预约</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detailBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>办理业务</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adviceBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>图书馆信箱</string>
    </property>
   </widget>
   <widget class="QPushButton" name="noticeBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>图书馆通知</string>
    </property>
   </widget>
   <widget class="QLabel" name="header">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="borrowBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>借阅情况</string>
    </property>
   </widget>
   <widget class="QPushButton" name="myreserveBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>我的预约</string>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>1000</width>
     <height>580</height>
    </rect>
   </property>
   <widget class="QWidget" name="page"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
